CHANGELOG.md

```diff
@@ -7,4 +7,23 @@ First public release, converts VCF variants into a [FHIR Genomics Diagnostic Rep
 * Supports VCF file (text-based or bgzipped) and optionally tabix files for query.

```

Comment:
We would have to correct this ChangeLog. Can you create two commits; the first one fixes the changelog basically changes #v0.1.0 to #0.0.1 - v0.0.16 
```
Converts VCF variants into a [FHIR Genomics Diagnostic Report](http://hl7.org/fhir/uv/genomics-reporting/index.html). 

### Additions:
...

```
The second commit adds Changelog for `v0.0.17`.

You can take a look at https://keepachangelog.com/en/1.0.0/ and read more about how to maintain a changelog. Can you please help includes details according to this?